app/channels/game_channel.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class GameChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from 'game_channel'
+    stream_from "game_channel_user_#{current_user.id}"
```

Comment:
Since it works not advocating any change just was curious if line 5 was redundant?

Reply:
I left it there in case there was anything we wanted to broadcast to all users of a game. It can be removed if you don't think it's necessary.

Reply:
Ahh excellent. Might want to use that for castling not for sure. Though I guess for castling since it reloads the screen of the player that did the castling could just have it do the same as the update. Not going to mess around with now.